pcl_conversions/package.xml

```diff
@@ -15,20 +18,29 @@
   <url type="repository">https://github.com/ros-perception/perception_pcl</url>
   <url type="bugtracker">https://github.com/ros-perception/perception_pcl/issues</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <build_export_depend>eigen</build_export_depend>
+  <build_depend>eigen3</build_depend>
+  <build_depend>libpcl-all-dev</build_depend>
+  <build_depend>message_filters</build_depend>
+  <build_depend>pcl_msgs</build_depend>
+  <build_depend>rclcpp</build_depend>
+  <build_depend>sensor_msgs</build_depend>
+  <build_depend>std_msgs</build_depend>
+
+  <build_export_depend>eigen3</build_export_depend>
   <build_export_depend>libpcl-all-dev</build_export_depend>
+  <build_export_depend>message_filters</build_export_depend>
   <build_export_depend>pcl_msgs</build_export_depend>
-  <build_export_depend>roscpp</build_export_depend>
+  <build_export_depend>rclcpp</build_export_depend>
   <build_export_depend>sensor_msgs</build_export_depend>
   <build_export_depend>std_msgs</build_export_depend>
 
-  <test_depend>eigen</test_depend>
-  <test_depend>libpcl-all-dev</test_depend>
-  <test_depend>pcl_msgs</test_depend>
-  <test_depend>roscpp</test_depend>
-  <test_depend>sensor_msgs</test_depend>
-  <test_depend>std_msgs</test_depend>
+  <exec_depend>libpcl-all-dev</exec_depend>
```

Comment:
They're all exec depends, if you want to simplify I'd support that. If you use the `<depend>` tag, it'll do  build, export, and execution dependency -- 1 liner to do all that above